src/test/resources/ValidOutputFile.csv

```diff
@@ -0,0 +1,3 @@
+1, 2, 3
+4, 5, 6
+7, 8, 9
```

Comment:
add new line

Reply:
This is fileWriter test, actual report creation is in FruitService and the new line generates there.